AcTransitMap/Views/Map/Index.cshtml

```diff
@@ -1,7 +1,5 @@
-﻿@*
-    For more information on enabling MVC for empty projects, visit https://go.microsoft.com/fwlink/?LinkID=397860
-*@
-@model IEnumerable<AcTransitMap.Models.VehiclePosition>
+﻿@model IEnumerable<AcTransitMap.Models.VehiclePosition>
+
 <style>
     #map-canvas {
         height: 500px;

```

Comment:
Change this to scale with the page height; e.g. `height: 100%` or something similar.